sv.c

```diff
@@ -17878,6 +17876,47 @@ Perl_sv_regex_global_pos_clear(pTHX_ SV *sv)
         mg->mg_len = -1;
 }
 
+/*
+=for apidoc sv_vstring_get
+
+If the given SV has vstring magic, stores the string pointer and length into
+the variables addressed by C<pvp> and C<lenp>, and returns true.  If not,
+returns false.
+
+=cut
+*/
+
+bool
+Perl_sv_vstring_get(pTHX_ SV * const sv, const char **pvp, STRLEN *lenp)
+{
+    PERL_ARGS_ASSERT_SV_VSTRING_GET;
+
+    MAGIC *mg = SvVSTRING_mg(sv);
+    if(!mg) return false;
+
+    if(pvp)  *pvp  = mg->mg_ptr;
+    if(lenp) *lenp = mg->mg_len;
+    return true;
+}
+
+/*
+=for apidoc sv_vstring_set
+
+Applies vstring magic to the given SV, storing the string given by C<pv> for
+C<len> bytes into it.
+
+=cut
+*/
+
+void
+Perl_sv_vstring_set(pTHX_ SV * const sv, const char *pv, STRLEN const len)
```

Comment:
Why isn't Perl_sv_vstring_set() sanitizing the user's input args `const char *pv`, `STRLEN len`? Why or why not, are both good answers.

Reply:
I don't follow the question at all.